.github/workflows/test.yaml

```diff
@@ -0,0 +1,74 @@
+name: Test
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9, 3.10-dev]
+      max-parallel: 1
+
+    steps:
+      - name: Print Github Context
+        env:
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+        run: echo $GITHUB_CONTEXT
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+
+      - name: pycache
+        uses: actions/cache@v2
+        with:
+          path: .cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
```

Comment:
All dependency packages are defined at setup.py. How about specifying it instead?